src/main/kotlin/com/hibob/project/dao/models.kt

```diff
@@ -52,4 +52,15 @@ data class Employee(
     val role: Role,
     val companyId: Long,
     val department: String
+)
+
+data class LoginEmployeeRequest(
+    val id: Long,
+    val companyId: Long
+)
+
+data class LoginEmployeeResponse(
```

Comment:
it's confusing if you call it Response and then use it as a param - maybe LogedInEmployee

or just employee is also ok if you don't have a separate employee data class for other things